test/Strategy/NullableStrategyTest.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/**
+ * @see       https://github.com/laminas/laminas-hydrator for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-hydrator/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-hydrator/blob/master/LICENSE.md New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace LaminasTest\Hydrator\Strategy;
+
+use Laminas\Hydrator\Strategy\NullableStrategy;
+use Laminas\Hydrator\Strategy\StrategyInterface;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers \Laminas\Hydrator\Strategy\NullableStrategy
+ */
+class NullableStrategyTest extends TestCase
+{
+    /**
+     * @var bool
+     */
+    protected $backupStaticAttributes = false;
+
+    /**
+     * @var bool
+     */
+    protected $runTestInSeparateProcess = false;
```

Comment:
Are these options needed for this test class?

Reply:
Without these options Psalm check could not be passed. May be Psalm need some tweeks.

Reply:
I removed these options. I dont understand why Psalm complains about my tests.